README.md

```diff
@@ -54,7 +54,49 @@ env, err := cel.NewEnv(
 
 That's it, the environment is ready to be use for parsing and type-checking.
 CEL supports all the usual primitive types in addition to lists, maps, as well
-as first-class support for JSON and Protocol Buffers.
+as first-class support for JSON and Protocol Buffers. More details on adding
+Protocol Buffers are below.
+
+#### Support for Protocol Buffers
```

Comment:
The README.md is intended to be a somewhat high-level view of CEL. Since the feature being used here is more of a best practice, I would recommend doing two things:

1. Create a wiki within the cel-go repo on how to work with protocol buffers.
2. Provide a [Go example test](https://blog.golang.org/examples) in addition to, or instead of the current test.

The nice part about a Go Example test is that it appears in the Go Docs and that it is executed and maintained as part of the standard test harness. In the future, I intend to add more Examples of how to use the various features in the `options.go` file.